src/test/resources/fftogff3_rules/multiple_entries1.embl

```diff
@@ -0,0 +1,26 @@
+ID   CACUVO010000001; SV 1; linear; genomic DNA; WGS; ENV; 59081 BP.
+XX
+AC   CACUVO010000001;
```

Comment:
Suggestion to rename the tests to be more meaningful
multiple_entries1.embl -> multiple_entries_with_features.embl